portfolio/src/main/webapp/charts.js

```diff
@@ -0,0 +1,74 @@
+google.charts.load("current", {
+  packages: ["geochart"],
+  mapsApiKey: "AIzaSyC35R1Q1qkQndHm-Ni6isXLbsSCQif3Umg"
+});
+google.charts.setOnLoadCallback(getChart);
+
+/**
+ * Gets the appropriate chart given the user selection from the dropdown.
+ */
+function getChart() {
+  const chartType = document.getElementById("chart-type").value;
+  if (chartType === "GeoMap") {
+    drawMarkersMap();
+  } else if (chartType === "LineChart") {
+    drawChart();
+  }
+}
+
+/**
+ * Create map that includes Japan's city and population information using GeoMap, which makes use of Google Charts API,
+ * Maps API, and Geocoding API.
+ */
+function drawMarkersMap() {
+  fetch("/japan-population")
+    .then(response => response.json())
+    .then(cityPopulations => {
+      const data = new google.visualization.DataTable();
+      data.addColumn("string", "City");
+      data.addColumn("number", "Population");
+      Object.keys(cityPopulations).forEach(city => {
+        data.addRow([city, cityPopulations[city]]);
+      });
+
+      const options = {
+        title: "Population of Japan's Largest Cities",
+        region: "JP",
+        displayMode: "markers",
+        colorAxis: { colors: ["blue", "red"] }
```

Comment:
https://google.github.io/styleguide/jsguide.html#features-objects-use-trailing-comma

There should be a comma after this object.